Expensify/App
# [No QA] Simplify electron-builder configs

### Details
Following up on https://github.com/Expensify/App/pull/7665 to do some cleanup.

### Fixed Issues
n/a

### Tests
1. Run `npm run desktop`, and verify that the app loads correctly.
1. Make a visible change to a React Native file, then save. The desktop app should hot-reload.
1. Run `npm run desktop-build-local`, install the app and make sure it works! It should have a production app icon, not a staging app icon.
1. Run `npm run desktop-build-staging-local`, install the app and make sure it works! It should have a staging app icon, not a production app icon.
1. Merge this PR and pray that it doesn't break our desktop deploys 🙃 


### QA Steps
None.


### Tested On


### Screenshots
#### Desktop (production build)
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/47436092/153640508-59ef93e1-d0af-4195-b6c1-faf2488a8e86.png">
<img width="541" alt="image" src="https://user-images.githubusercontent.com/47436092/153640593-937ffe5e-0492-4377-8757-b93958e42663.png">
<img width="415" alt="image" src="https://user-images.githubusercontent.com/47436092/153647973-08fdaa1e-218a-431a-bfda-2e527fdb179a.png">

#### Desktop (staging build)
<img width="445" alt="image" src="https://user-images.githubusercontent.com/47436092/153661240-df951b1a-0289-4ee1-b350-78e851a261eb.png">
